control_panel_api/tests/__init__.py

```diff
@@ -1,5 +1,8 @@
 from django.conf import settings
 
+from control_panel_api.tests.filters import *
+from control_panel_api.tests.permissions import *
```

Comment:
Don't do this `import *` it is an antipattern. You must use explicit imports.

Reply:
Should I import like this instead?

```python
import control_panel_api.tests.filters
import control_panel_api.tests.permissions
```

Reply:
Requiring individual testcase imports is going to increase the chance of tests being written and not executed.

So it's a toss up of not restructuring them, or `import *` I'd say.

Reply:
Changed: daf525ae12e33a09defa53e1a26324777f581859